### ISSUE TITLE
Add Tanzania to Map Collection

### ISSUE THREAD

**Issue by BossBele** [VictorCazanave/svg-maps#53]
**Is your feature request related to a problem? Please describe.**
Currently, Tanzania is not implemented.

**Describe the solution you'd like**
Tanzania should be added to the list. I'm here to show that there's a demand for it.

**Additional context**
Ref.: https://en.wikipedia.org/wiki/Tanzania
         https://simplemaps.com/resources/svg-tz

I will also be willing to provide necessary support to see this through.
I am not very knowledgeable to create my own svg and therefore I ask the community to contibute.

**Comment by VictorCazanave** [comment]
Thank you for opening this issue @BossBele !
I'd be glad to add Tanzania map, but unfortunately [simplemaps license](https://simplemaps.com/resources/svg-license) doesn't allow to copy and distribute their maps.
Do you know another good source for the SVG?

**Comment by BossBele** [comment]
That's unfortunate.
I looked up couple of other sites:
https://vemaps.com/tanzania/tz-04 - Requires Attribution
https://upload.wikimedia.org/wikipedia/commons/d/df/Tanzania_DarEsSalaam_location_map.svg - Seems to be out of standards
https://www.amcharts.com/svg-maps/?map=tanzania - Requires Attribution (https://www.amcharts.com/online-store/)

That's all I could find. Let me know if they can be of use.

**Comment by VictorCazanave** [comment]
Thanks @BossBele !
The map of amcharts.com looks good. If I have time, I'll add it next week.

**Comment by BossBele** [comment]
> Thanks @BossBele !
> The map of amcharts.com looks good. If I have time, I'll add it next week.

Sounds Great!

**Comment by BossBele** [comment]
@VictorCazanave Any progress?

**Comment by VictorCazanave** [comment]
Sorry for the lack of feedback!

Actually I started to work on the map of amcharts.com a while ago, but I was confused by the different names and number of regions (compared to [wikipedia](https://en.wikipedia.org/wiki/Regions_of_Tanzania)), then I was too busy to continue 😅  

I finally figured out how to rename the regions properly. However it looks like the map is outdated because it doesn't contain [Songwe Region](https://en.wikipedia.org/wiki/Songwe_Region) which was created in 2016.
I'll still publish it today.
